### ISSUE TITLE
Create module of dielectric functions

### ISSUE THREAD

**Issue by lemmatum** [PlasmaPy/PlasmaPy#214]
We should make a module of dielectric (susceptibility) functions. These are super useful for things like simulating Thomson scattering spectra. 

I already have some code for this, so I could get it started. Are there any specific dielectric functions we should include?

**Comment by lemmatum** [comment]
[Here](https://arachnoid.com/latex/?equ=%5Cepsilon(%5Comega%2C%5Cvec%7Bk%7D)%20%3D%20v_%7Bphase%7D%5E2%20%5Cint_%7B-%5Cinfty%7D%5E%7B%2B%5Cinfty%7D%20%5Cfrac%7B%5Cvec%7Bk%7D%20%5Ccdot%20%5Cfrac%7B%5Cpartial%20f(%5Cvec%7Bv%7D)%7D%7B%5Cpartial%20%5Cvec%7Bv%7D%7D%7D%20%7B%20(%5Comega%20-%20%5Cvec%7Bk%7D%20%5Ccdot%20%5Cvec%7Bv%7D%20-%20i%20%5Cgamma)%7D%20d%20%5Cvec%7Bv%7D) is one form dielectric function I would like to add. I think it is from Landau's Physical Kinetics, but can be found in a number of plasma physics texts.

**Comment by jhillairet** [comment]
Is the module have been already created ? If not, where should it be created ? I've got few function for cold plasma dielectric tensor function (Stix parameters for instance)

**Comment by StanczakDominik** [comment]
I don't believe we have it yet. If you'd like to contribute those functions, `plasmapy/physics/dielectric.py` would probably be a reasonable place. We'd of course need to update the __init__ file in that directory to add that file.

**Comment by lemmatum** [comment]
I've not gotten around to creating it yet. You are welcome to start it!
I agree `plasmapy/physics/dielectric.py` is a good name/place.

**Comment by namurphy** [comment]
We've made a good amount of progress on this in `dielectric.py`.  What else do we need to do before we can close this issue?
